scripts/comparison/classes.py

```diff
@@ -0,0 +1,200 @@
+import os
```

Comment:
change the name of the file from classes to something meaningfull: like  html_utils.py